src/components/Search/style.js

```diff
@@ -1,22 +1,23 @@
-import style from 'styled-components';
-import { HotKeys } from 'react-hotkeys';
+import styled from 'styled-components';
 
-export const SearchWrapper = style(HotKeys)`
+export const SearchWrapper = styled.div`
   height:  auto;
 `;
 
-export const SearchInput = style.input`
-    border: 0;
-    border-bottom:  1px solid #ecf0f3;
-    border-radius: 4px;
-    box-shadow: none;
-    width: 100%;
-    padding: 0 7px;
-    box-sizing: border-box;
-    resize: none;
-    font-size: 14px;
-    height: 35px;
-    &:focus {
-      outline: none;
+export const SearchInput = styled.input`
+    && {
+      border: 0;
```

Comment:
we're adding the `&&` to override any existing input styles in apps.

Reply:
this is awesome! Something I was going to take a look at inside of Reply, very happy to see it here!